client/e2e/utils/queries.ts

```diff
@@ -0,0 +1,87 @@
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./e2e/.env.local" });
+
+export const upsertOperator = {
+  text: `
+    INSERT INTO erc.operator (id, status, legal_name, trade_name, cra_business_number, bc_corporate_registry_number, business_structure_id, is_new)
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
+    ON CONFLICT (id)
+    DO UPDATE SET status = EXCLUDED.status;
+  `,
+  values: [
+    2,
+    "Approved",
+    "Existing Operator 2 Legal Name",
+    "Existing Operator 2 Trade Name",
+    "987654321",
+    "def1234567",
+    "BC Corporation",
+    false,
+  ],
+};
+
+export const upsertOperatorUser = {
+  text: `
+     INSERT INTO erc.user (user_guid, app_role_id, first_name, last_name , position_title , email, phone_number, business_guid, bceid_business_name )
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
+    ON CONFLICT (user_guid)
+    DO UPDATE SET app_role_id = EXCLUDED.app_role_id
+  `,
+  values: [
+    process.env.E2E_INDUSTRY_USER_ADMIN_GUID as string,
+    "industry_user",
+    "Cas",
```

Comment:
nit: We could use more meaningful mock data (e.g., a real name instead of "Cas," something that looks more like a real operator name vs. "Operator 1" to make it easier to debug

Reply:
I believe that name is returned from Keycloak for that guid: Bcgov Cas